components/generic/HeroBanner.vue

```diff
@@ -0,0 +1,18 @@
+<template>
```

Comment:
Let's add a <script> block already with the hard-coded values as defaults for props.